config/config.go

```diff
@@ -155,6 +155,8 @@ type Config struct {
 	// download files to. If FileOpenSaveFilesPermanently has been set to
 	// true, then all opened files are saved in this folder for example.
 	FileDownloadSaveLocation string
+
+	VimEnabled bool
```

Comment:
Comment what exactly this entails when it is set to true.

Reply:
Will do